.travis.yml

```diff
@@ -19,7 +19,7 @@ cache:
     - $HOME/.phpbrew
 
 before_install:
-  - nvm install 6.11.1 && nvm use 6.11.1
+  - nvm install && nvm use
   - npm install --global npm@5
```

Comment:
Do we still need to override npm on Travis?

Reply:
It is probably safe to drop this, since Node 8.x will come with npm 5.x. It may be slightly different behavior in that this would install the latest 5.x, whereas I expect Node will come preinstalled with a pinned version of npm. I don't think we're specifically relying on this though.

Reply:
npm's policy is that the latest release of npm is the supported release regardless of any _pinned version_ a Node.js release may ship.